src/components/accounts/EnterVerificationCode.js

```diff
@@ -79,24 +80,20 @@ const SetRecoveryMethodSuccess = ({
                     disabled={code.length !== 6 || loading}
                     sending={loading}
                 >
-                    <Translate id='button.confirm' />
+                    <Translate id='button.verifyCodeEnable' />
                 </FormButton>
             </form>
-            <div className='re-enter'>
-                <Translate id={`setRecoveryConfirm.reenter.one.${useEmail ? 'email' : 'phoneNumber'}`} />
-                <span onClick={onGoBack} className='link'><Translate id='setRecoveryConfirm.reenter.link'/></span>
-                <Translate id={`setRecoveryConfirm.reenter.two.${useEmail ? 'email' : 'phoneNumber'}`} />
-            </div>
+            <p><Translate id='setRecoveryConfirm.didNotRecive'/> <span onClick={onResend} className='link'><Translate id='setRecoveryConfirm.resendCode'/></span>, <Translate id='setRecoveryConfirm.or'/> <span onClick={onGoBack} className='link'><Translate id='setRecoveryConfirm.sendToDifferent'/></span>.</p>
```

Comment:
Why collapse everything into single line? It was much easier to read before change.